### ISSUE TITLE
After setting up the server, phone can access Internet, but cannot login into the game. 

### ISSUE THREAD

**Issue by austinxw** [justinleewells/pogo-optimizer#168]
Mac user. OX 10.11.6. Xcode 8.3 beta.

It is not the problem of Niantic, because I can login into the game without optimizer. 

I am pretty sure I have settle down the certificate and the proxy configuration, since I can access the Internet (i.e., google.com) without any problem.  

The following text shows the log when I start "npm start"

`> pogo-optimizer@1.2.0 start /Users/Ding/Documents/pokemon/pogo-optimizer
> gulp && node index

(node:7870) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
[21:29:01] Using gulpfile ~/Documents/pokemon/pogo-optimizer/gulpfile.js
[21:29:01] Starting 'clean'...
[21:29:02] Finished 'clean' after 244 ms
[21:29:02] Starting 'jade'...
[21:29:02] Starting 'stylus'...
[21:29:02] Starting 'img'...
[21:29:02] Starting 'lib-js'...
[21:29:02] Starting 'lib-css'...
[21:29:02] Starting 'favicon'...
[21:29:02] Starting 'js'...
[21:29:02] Starting 'font'...
[21:29:02] Finished 'favicon' after 455 ms
[21:29:02] Finished 'lib-css' after 476 ms
[21:29:02] Finished 'stylus' after 732 ms
[21:29:03] Finished 'lib-js' after 854 ms
[21:29:03] Finished 'js' after 865 ms
[21:29:03] Finished 'jade' after 923 ms
[21:29:03] Finished 'font' after 1.21 s
[21:29:04] Finished 'img' after 2.2 s
[21:29:04] Starting 'default'...
[21:29:04] Finished 'default' after 6.57 μs
[+++] PokemonGo MITM Proxy listening on 8081
[!] Make sure to have the CA cert .http-mitm-proxy/certs/ca.pem installed on your device
Pokemon GO Optimizer front-end listening on port 3000
Accept the cert at 192.168.147.122:3000/ca.pem if you haven't already done so
WARNING: Cannot set inventory. No player logged in.

`

**Comment by justinleewells** [comment]
Just close the game and re-open it. It has to intercept the PlayerData message.

**Comment by austinxw** [comment]
I tried, I cannot access into the game. It gets stuck. 

"WARNING: Cannot set inventory. No player logged in." This message may because that, I first login into the game without optimizer, then change to wifi. 

Anyway, I tried several times to access the game with optimizer... 

Actually it works the day before yesterday. I don't know why it does not work today.

**Comment by OSwoboda** [comment]
I have the same problem. I've set everything up, but can't login while proxy is active.

**Comment by austinxw** [comment]
Actually, I also help my friend to settle up the server remotely. He also faced the same problem...
@OSwoboda

**Comment by justinleewells** [comment]
It's interesting that your computer is cool with intercepting messages after you've logged in. I'll look into it a bit tonight, but I can't really reproduce it. Any chance you're pretty solid with computers and could try setting up https://github.com/rastapasta/pokemon-go-mitm-node to see if that allows you to log in?

**Comment by therealklanni** [comment]
@austinxw have you tried completely killing the app and reopen it?

**Comment by austinxw** [comment]
@justinleewells sorry I cannot set the one you suggested, since I need to catch a flight for traveling soon. 
I can use this tool perfectly the day before yesterday (the port for the website was 5000). What I did these two days to my computer is that I upgraded the Xcode to 8.3 beta. Does this behavior changes something?

**Comment by austinxw** [comment]
@therealklanni yes, sure. I tried several times. If no one met this problem, maybe I will try to reinstall the Optimizer after the traveling. Anyway, It is a really good tool for trainers!! Thanks

**Comment by KerekesDavid** [comment]
@austinxw I had the same problem, turns out I installed the certificate for the brower on my phone (firefox) only. I had to redownload the cert using the default browser on android, then select the use for VPN-s option. Now it works like a charm.

**Comment by austinxw** [comment]
@KerekesDavid I use iPhone 6p, and I think it is not the problem with certificate. (I did the exact same procedure the day before yesterday, and it works).

**Comment by danieledifice** [comment]
did you tried to redownload the certificate?

**Comment by fbatista** [comment]
I noticed today that the certificate expires really fast, and that just re-downloading from the /ca.pem or /ca.crt won't work, because if the server is not restarted, it will still download the same outdated version.

**Comment by justinleewells** [comment]
Niantic implemented SSL pinning, so proxying no longer works.
